### PR TITLE
SLR-1036 part2

### PR DESCRIPTION
**Pull request by arijit100**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SLR-1036

### Change description ###
Added working patterns and payment type

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] No
```

### CODE REVIEW

File: features/references/manage_payment_type.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# FeatureID  EpicID StoryID TestID
+Feature: Judicial Office Holder as both Salaried and Fee Paid
+
+  As a Solution
+  I want to be able to identify if a judicial office holder is both salaried and fee paid,
+  So that It can be identified when the judicial office holder is allocated to a hearing as 'Salaried' or 'Fee Paid'.
+
+  Background:
+    Given Hearing are setup
```
Comment (bugbare-reform) [inline features/references/manage_payment_type.feature:9]: GIVEN a hearing for tomorrow is created
AND a Judicial Holder is created

File: features/references/manage_payment_type.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  @pending
+  @manage_references
+  @manage_payment_type
+  @SLR-330 @SLR-600 @SLR-899 @SLR-798
+  Scenario: Identify Judicial Office Holder as both Salaried and Fee Paid
+    Given I have a existing multi-day hearing
```
Comment (bugbare-reform) [inline features/references/manage_payment_type.feature:16]: should not need to be multi-day - right?

File: features/references/manage_retirement.feature [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# FeatureID  EpicID StoryID TestID
+Feature: No Hearings after Judicial Office Holder Retirement Date
+
+  As a System Admin
+  I want to be able to set the retirement date of a judicial office holder
+  So that the judicial office holder is not allocated to a hearing beyond the retirement date.
+
+  Background:
+    Given Hearing are setup
```
Comment (bugbare-reform) [inline features/references/manage_retirement.feature:9]: I think it's better to say GIVEN a Hearing for <date> is Created as this is better reuse of an operation that will definitely be needed everywhere.
You also need to state that JOH's are created if you are setting everything up in the background

File: features/references/manage_working_patterns.feature [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# FeatureID  EpicID StoryID TestID
+Feature: Identify Judicial Office Holder Working Patterns and allocate to hearing that they are working and available
+
+  As a Solution
+  I want to be able to identify a judicial office officer's daily and weekly working pattern.
+  So that the hours worked each day and the days of the week worked can be applied to the scheduling and listing of hearings.
+
+  Background:
+    Given Hearing are setup
```
Comment (bugbare-reform) [inline features/references/manage_working_patterns.feature:9]: as mentioned in previous file reviews, update this statement and add JOH creation step too

File: features/references/manage_working_patterns.feature [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# FeatureID  EpicID StoryID TestID
```
Comment (bugbare-reform) [inline features/references/manage_working_patterns.feature:1]: I think that the premise of matching means that you need to verify the JOH working pattern, then go to the allocation page for a hearing that is scheduled and listed for a specific day - if the hearing is scheduled for a day / hour that matches the pattern then the JOH should be allocated, if not then you should not be able to allocate the JOH to that hearing

Comment (bugbare-reform) [reply features/references/manage_working_patterns.feature:1]: Given I am on judicial office holder Page
    And   the judicial office holder works Monday to Friday
    And  the judicial office holder is Full Time
    When  I view a hearing that is on Monday within standard working hours
    Then  the judicial office holder can only be assigned to the matching hearing

File: features/references/manage_working_patterns.feature [hunk 6]
```diff
@@ -0,0 +1,57 @@
+Feature: Identify Judicial Office Holder Working Patterns and allocate to hearing that they are working and available
+
+  As a Solution
+  I want to be able to identify a judicial office officer's daily and weekly working pattern.
+  So that the hours worked each day and the days of the week worked can be applied to the scheduling and listing of hearings.
+
+  Background:
+    Given Multi-Day Hearing is created
```
Comment (bugbare-reform) [inline features/references/manage_working_patterns.feature:8]: You would also need a single full day and single partial day hearing

Comment (arijit100) [reply features/references/manage_working_patterns.feature:8]: Updated the background to have a single and partial day hearing

**Pull request merged**